_config.yml

```diff
@@ -68,6 +68,9 @@ sidebar_categories:
     - docs/advanced/features/external-logins
     - docs/advanced/features/return-merchandise-authorization
     - docs/advanced/features/in-stock-alert
+    - docs/advanced/features/maintenance-mode
+    - docs/advanced/features/disable-hyration
```

Comment:
A link to this page should also be added to https://developers.front-commerce.com/docs/advanced/theme/server-side-rendering.html#Learn-more for discoverability